stemcell_builder/stages/static_libraries_config/apply.sh

```diff
@@ -5,14 +5,23 @@ set -e
 base_dir=$(readlink -nf $(dirname $0)/../..)
 source $base_dir/lib/prelude_apply.bash
 source $base_dir/lib/prelude_bosh.bash
+source $base_dir/etc/settings.bash
 
 cp -p "${assets_dir}/${DISTRIB_CODENAME}_static_libraries_list.txt" $chroot/var/vcap/bosh/etc/static_libraries_list
 
 
+kernel_suffix="-generic"
 if [[ "${DISTRIB_CODENAME}" == 'bionic' ]]; then
     major_kernel_version="5.4"
 else
     major_kernel_version="4.15"
 fi
-kernel_version=$(find $chroot/usr/src/ -name "linux-headers-$major_kernel_version.*-generic" | grep -o '[0-9].*-[0-9]*-generic')
-sed -i "s/__KERNEL_VERSION__/$kernel_version/g" $chroot/var/vcap/bosh/etc/static_libraries_list
\ No newline at end of file
+
+if [[ "${stemcell_operating_system_variant}" == 'fips' ]]; then
+    kernel_suffix="-aws-fips"
```

Comment:
shouldn't that be the `iaas` variant? or at least exit if that variant isn't `aws` ?